src/Illuminate/Database/Console/Migrations/MigrateMakeCommand.php

```diff
@@ -18,7 +19,8 @@ class MigrateMakeCommand extends BaseCommand
         {--table= : The table to migrate}
         {--path= : The location where the migration file should be created}
         {--realpath : Indicate any provided migration file paths are pre-resolved absolute paths}
-        {--fullpath : Output the full path of the migration}';
+        {--fullpath : Output the full path of the migration}
+        {--P|primary=int : Set default primary key type for the table <info>[int, uuid, ulid]</info>}';
```

Comment:
`--key` might be shorter but I don't find it as descriptive as `--primary`. Thoughts?